src/components/needle.js

```diff
@@ -1,12 +1,13 @@
-import React from 'react';
-import d3Shape from "d3-shape";
+import React from "react";
 
 export default class Needle extends React.Component {
+
   render() {
+    const {rotation, height} = this.props;
     return (
-      <g transform={`rotate(50)`}>
+      <g transform={`rotate(${rotation})`}>
         <path
-          d="M 0 5 C -1,5 -4,3 -6,0 L 0 -175 L 6 0 C 4,3 1,5 0,5"
+          d={`M 0 5 C -1,5 -4,3 -6,0 L 0 -${height} L 6 0 C 4,3 1,5 0,5`}
```

Comment:
code smell, no hard coded values. needle should be configureable, and then these should be path values should be calculated. If that's coming later that's fine, but extract this into a method and add a todo